### DISCUSSION TITLE
增加Qexo内的CDN链接

### DISCUSSION THREAD

**Discussion by Fgaoxing** [Qexo/Qexo#383] [Show and tell]
可以吧这个加到Qexo的列表里吗？https://cdn2.chuqis.com/cdnjs

**Comment by Fgaoxing** [comment]
这是位于初七云的CDNN，目前还不支持NPM，不过不影响CDNJS加不加ajax/libs都行，速度还行，稳定性在测，另外也是比较安全的，文件都会进行校验，所以不用担心篡改

**Comment by am-abudu** [comment]
接下来会全面剔除CDNJS。因为CDNJS的Packages更新Bug太多了

**Reply by Fgaoxing** [reply]
OK，那你用啥？需要我提供那个的CDN？
